src/scancode/api.py

```diff
@@ -168,7 +169,7 @@ def get_licenses(
     from licensedcode import cache
     from licensedcode.spans import Span
 
-    idx = cache.get_index()
+    idx = cache.get_index(additional_directories=additional_directories)
```

Comment:
The index should be:
- not existing in the cache in which case we could either rebuild or we could simply error out asking for a reindex.
- existing in the cache and used as-is with whatever directories was used to create it....but passing down additional_directories should never be needed.